### PR TITLE
Fix .NET Core 3.1 and 6.0 starter

### PR DESCRIPTION
**Pull request by adityaoberai**

### CODE REVIEW

File: dotnet-3.1/src/Index.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@
   
   If an error is thrown, a response with code 500 will be returned.
 */
+using System.Threading.Tasks;
```
Comment (adityaoberai) [inline dotnet-3.1/src/Index.cs:13]:
```suggestion

using System.Threading.Tasks;
```

Just a little extra addition

**Review by Meldiron — APPROVED** [review]

**Comment by Meldiron** [comment]
Did manual QA, works.

**Comment by Meldiron** [comment]
@adityaoberai can we do the same in 6.0? Or not necessary?

**Comment by adityaoberai** [comment]
I'll make the change

**Comment by adityaoberai** [comment]
@Meldiron I've done manual QA and added the using directives to the .NET 6.0 function starter too

**Comment by adityaoberai** [comment]
@christyjacob4 can we merge this? It's been open for a while 😅

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**